src/adapter/src/catalog.rs

```diff
@@ -4003,12 +4093,41 @@ impl<S: Append> Catalog<S> {
             )
             .await
             .expect("cannot fail to allocate system ids");
-        ConcreteComputeInstanceReplicaLogging::Concrete(
+
+        DEFAULT_LOG_VIEWS
+            .clone()
+            .into_iter()
+            .zip(system_ids.into_iter())
+            .collect()
+    }
+
+    /// Allocate ids for persisted logs. Called once per compute replica creation
+    pub async fn allocate_persisted_introspection_source_indexes(
```

Comment:
The name of this method seems like it's might not be as accurate anymore? I *think* these IDs are for sources and views, not indexes?

Reply:
I changed the names there are now three functions:
 * allocate_arranged_introspection_sources
 * allocate_persisted_introspection_views
 * allocate_persisted_introspection_items

and i also changed the local var names + Action and Op members